src/redis.cr

```diff
@@ -43,6 +43,7 @@ class Redis
   # :nodoc:
   alias Request = Array(RedisValue)
 
+  getter! location : String
```

Comment:
Thanks for the PR!

Is there a reason that you call the method `location` instead of `url` as in redis-rb? Otherwise I would like to call it `url`.